integration/simulation/simulation.go

```diff
@@ -41,19 +41,15 @@ func (s *Simulation) Start() {
 
 	timer := time.Now()
 	go s.TxInjector.Start()
-
-	stoppingDelay := 2 * time.Second
-
 	// Wait for the simulation time
-	time.Sleep(s.SimulationTime - stoppingDelay)
+	time.Sleep(s.SimulationTime)
 
 	s.TxInjector.Stop()
 
 	// allow for some time after tx injection was stopped so that the network can process all transactions
-	time.Sleep(stoppingDelay)
+	time.Sleep(10 * time.Second)
```

Comment:
10 seconds feels like quite a lot. Weren't 2 seconds enough?

Reply:
Still trying to figure out what's happening.

Seems like the nodes need more wait time to finish issuing the rollups, otherwise the efficiencies fail by a lot.
I can't correctly reproduce it locally, seems to only happen on GH.